__tests__/Car.test.js

```diff
@@ -0,0 +1,42 @@
+import Car from '../src/main';
+
+describe('자동차 경주 테스트', () => {
+  const CAR_NAME = 'Bentley';
+  const INITIAL_LOCATION = 0;
+
+  it('자동차는 이름을 가지고 있다.', () => {
+    const car = new Car(CAR_NAME);
+    expect(car.name).toBeDefined();
+  });
+
+  it('자동차에 이름을 부여할 수 있다', () => {
+    const car = new Car(CAR_NAME);
+    expect(car.name).toEqual(CAR_NAME);
+  });
+
+  it('자동차는 위치 값을 가지고 있다.', () => {
+    const car = new Car(CAR_NAME);
+    expect(car.location).toBeDefined();
+  });
+
+  it('자동차 위치의 초기 값은 0이다.', () => {
+    const car = new Car(CAR_NAME);
+    expect(car.location).toEqual(INITIAL_LOCATION);
+  });
+
+  it('자동차는 전진할 수 있어야 한다.', () => {
+    const car = new Car(CAR_NAME);
+    expect(car.forward).toBeDefined();
+  });
+
+  it('자동차가 전진하는 경우 위치값이 1 증가한다.', () => {
+    const car = new Car(CAR_NAME);
+    const movingForward = jest.spyOn(car, 'forward');
+    const beforeForwardLocation = car.location;
+
+    car.forward();
+    expect(movingForward).toHaveBeenCalled();
+
+    expect(car.location).toEqual(beforeForwardLocation + 1);
```

Comment:
단순하게 car.forward() 실행하고, car.location 으로 검증하면 되지 않을까요?

```suggestion
    const car = new Car(CAR_NAME);
    expect(car.location).toEqual(0);
    car.forward();
    expect(car.location).toEqual(1);
```
요로코롬..